pwiz_tools/BiblioSpec/tests/reference/Small_Library-Positive-ions_CE-Merged.check

```diff
@@ -36,3 +36,4 @@ id	scoreType	probabilityType
 17	BYONIC SCORE	PROBABILITY_THAT_IDENTIFICATION_IS_INCORRECT
 18	PEPTIDE SHAKER CONFIDENCE	PROBABILITY_THAT_IDENTIFICATION_IS_CORRECT
 19	GENERIC Q-VALUE	PROBABILITY_THAT_IDENTIFICATION_IS_INCORRECT
```

Comment:
This is annoying. Do we even need to output this table for every check file, requiring that adding a new score changes every check file? If not then we can take the opportunity here to change all files by removing them all instead of changing all files adding one more row that is irrelevant to all the files except one.

Reply:
Yeah, Brendan and I were musing on this too. It doesn't seem very high value - all it really checks for is did somebody accidentally change something from ROBABILITY_THAT_IDENTIFICATION_IS_CORRECT to ROBABILITY_THAT_IDENTIFICATION_IS_INCORRECT etc.